LoopCore/LoopSettings.swift

```diff
@@ -181,16 +174,15 @@ extension LoopSettings {
         preMealOverride = nil
     }
 
-    public func legacyWorkoutOverride(beginningAt date: Date = Date(), for duration: TimeInterval) -> TemporaryScheduleOverride? {
+    public mutating func legacyWorkoutOverride(beginningAt date: Date = Date(), for duration: TimeInterval) -> TemporaryScheduleOverride? {
```

Comment:
Seems a bit odd to me to still have `LoopSettings` have this responsibility yet have `LoopSettingsManager` issue the reminder.  Do you agree?